pyproject.toml

```diff
@@ -35,6 +35,7 @@ qualang-tools = { version = "^0.15.0", optional = true}
 setuptools = { version = ">67.0.0", optional = true }
 laboneq = { version = "==2.25.0", optional = true }
 qibosoq = { version = ">=0.0.4,<0.2", optional = true }
+qutip = "^4.7.4"
```

Comment:
I would keep it optional, as it is for the other instruments.

Reply:
Fixed in 0ebe823